browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1236,7 +1236,6 @@ L.Control.JSDialogBuilder = L.Control.extend({
 					// Determine key direction
 					let key;
 					if (e.key === 'Tab') {
-						e.preventDefault(); // Always prevent default tab behavior
```

Comment:
is that   intentional? It was for recnet tabs focus switch.
I see no connection to split button

Reply:
I see it fixes jump form font selector to sidebar